integrity/typed_test.go

```diff
@@ -1753,3 +1759,66 @@ func TestTypedWatch(t *testing.T) {
 		driverMock.MinimockFinish()
 	})
 }
+
+func TestTypedGet_PassingModRevision(t *testing.T) {
```

Comment:
The test is too hard to understand:

It's not clear where the beginning is and where the end is. Where are which parts of AAA: Arrange, Act, and Assert.